api/db/changelog/db.changelog-225-add-user-discovery-source.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="Peter-Lavigne" id="changelog-225-add-user-discovery-source">
+    <addColumn tableName="user">
+      <column name="user_general_discovery_source_other_text" type="varchar(255)"/>
```

Comment:
is 255 enough? 
(context: I over-underestimated how much researcher want to type and had similar issue with another DB field: https://github.com/all-of-us/workbench/commit/7e7877c76416cbde9c9894802330c242340397a8 )

Reply:
I think 255 is OK, since this is in the context of a small checkbox description. The longest other option, `All of Us Evenings with Genetics Research Program, Baylor College of Medicine, Department of Molecular and Human Genetics`, is only 121 characters.

However, the UI was limiting users to only 100 characters, so I increased it to 250.